src/shared/treeview/nodes/awsTreeNodeBase.ts

```diff
@@ -7,6 +7,8 @@ import { TreeItem, TreeItemCollapsibleState, commands } from 'vscode'
 import { isCloud9 } from '../../extensionUtilities'
 
 export abstract class AWSTreeNodeBase extends TreeItem {
+    public regionCode: string | undefined
```

Comment:
I originally had this but changed it because it caused linter errors in `src/awsexplorer/regionNode.ts`, but I can get around it by replacing `regionCode` with `region.id`. They should always be equal so not sure why the linter was complaining. 

I agree, it seems dangerous to have it be able to be overwritten somewhere and no reason it should be.